### ISSUE TITLE
Telegram do not work

### ISSUE THREAD

**Issue by klinux** [grafana/oncall#137]
I'm trying to connect Telegram but I receive this error:

Telegram, because the incident is not posted to Telegram (reason: Telegram is disabled for the route)

The user was successfully connect to the bot, there something that I need to configure?

Thanks

**Comment by vadimkerr** [comment]
Hey @klinux, in order to receive messages from Telegram bot, you should connect a Telegram channel on the "ChatOps" page and then use it in the route settings on the "Integrations" page.

**Comment by Matvey-Kuk** [comment]
@vadimkerr may be let's the improve log message to make it more clear?

**Comment by vadimkerr** [comment]
@Matvey-Kuk yes, we should do that. We also could consider allowing posting to Telegram DMs without connecting a channel.

**Comment by Matvey-Kuk** [comment]
@vadimkerr I think we had it in our codebase, didn't we?

**Comment by vadimkerr** [comment]
@Matvey-Kuk we had it, but decided to enforce channel connection at some point. We could change this back to have DM messages without channel connection.

**Comment by klinux** [comment]
Hi everyone, thank you for the reply, well, I trying to **Connect Telegram Channel**, in the chatops tab, but when I copy the activation code, and paste it on channel, not happens, and when I click at Done button, the screen back to the button to connect telegram channel, and nothing in logs of oncall-engine.

Thank you

**Comment by vadimkerr** [comment]
This should be fixed as https://github.com/grafana/oncall/pull/142 is now merged and released. 
@klinux could you try this again on a new version of Grafana OnCall?

**Comment by Matvey-Kuk** [comment]
@vadimkerr does the PR fix the issue described above with bot being silent when user sends a code to the channel?

**Comment by klinux** [comment]
@vadimkerr @Matvey-Kuk 
Yes, works like a charm, thank you guys, DM works the channel work too with this fix, here everything is ok now.

Thank you

**Comment by Matvey-Kuk** [comment]
Thank you! Closing
